planning/behavior_velocity_run_out_module/src/utils.cpp

```diff
@@ -181,20 +181,15 @@ bool pathIntersectsEgoCutLine(
   const double half_line_length, std::vector<geometry_msgs::msg::Point> & ego_cut_lane)
 {
   if (path.size() < 2) return false;
-  const auto ego_rpy = rpyFromQuat(ego_pose.orientation);
-  const double ego_yaw = ego_rpy.z;
-
-  auto getOffsetPoint = [](
-                          const geometry_msgs::msg::Pose & ego_pose, const double half_line_length,
-                          const double angle) {
-    geometry_msgs::msg::Point p;
-    p.x = ego_pose.position.x + half_line_length * std::cos(angle);
-    p.y = ego_pose.position.y + half_line_length * std::sin(angle);
-    return p;
-  };
-
-  const geometry_msgs::msg::Point p1 = getOffsetPoint(ego_pose, half_line_length, ego_yaw + M_PI_2);
-  const geometry_msgs::msg::Point p2 = getOffsetPoint(ego_pose, half_line_length, ego_yaw - M_PI_2);
+  auto getOffsetPoint =
+    [](const geometry_msgs::msg::Pose & ego_pose, const double half_line_length) {
+      const auto p = tier4_autoware_utils::calcOffsetPose(ego_pose, 0.0, half_line_length, 0.0);
+      return tier4_autoware_utils::createPoint(p.position.x, p.position.y, p.position.z);
```

Comment:
```cpp
tier4_autoware_utils::calcOffsetPose(ego_pose, 0.0, half_line_length, 0.0).position
```
is enough.

Reply:
@takayuki5168 Ah! right thanks for the review